File: src/sdk-reference/cpp/1/server/get-stats/snippets/get-stats.cpp
try {
  time_t start = std::time(0);
  time_t stop = std::time(0);

  std::string stats = kuzzle->server->getStats(start, stop);

  std::cout << "Kuzzle Stats as JSON string: " << stats << std::endl;
} catch (kuzzleio::KuzzleException e) {
  std::cerr << e.getMessage() << std::endl;
}

File: src/sdk-reference/csharp/1/server/get-stats/snippets/get-stats.cs
try {
  time_t start = std::time(0);
  time_t stop = std::time(0);

  string stats = kuzzle.server.getStats(start, stop);

  Console.WriteLine("Kuzzle Stats as JSON string: " + stats);
} catch (KuzzleException e) {
  Console.Error.WriteLine(e.what());
}
